### PR TITLE
added logging

### PR DESCRIPTION
**Pull request by tarkaiev**

### CODE REVIEW

File: src/main/java/movie/tickets/Main.java [hunk 1]
```diff
@@ -15,9 +15,11 @@
 import movie.tickets.service.OrderService;
 import movie.tickets.service.ShoppingCartService;
 import movie.tickets.service.UserService;
+import org.apache.log4j.Logger;
 
 public class Main {
     private static Injector injector = Injector.getInstance("movie.tickets");
+    private static Logger logger = Logger.getLogger(Main.class);
```
Comment (Martynice) [inline src/main/java/movie/tickets/Main.java]: This field should be final

File: src/main/java/movie/tickets/dao/impl/CinemaHallDaoImpl.java [hunk 2]
```diff
@@ -6,21 +6,26 @@
...
 import movie.tickets.model.CinemaHall;
 import movie.tickets.util.HibernateUtil;
+import org.apache.log4j.Logger;
 import org.hibernate.Session;
 import org.hibernate.Transaction;
 import org.hibernate.query.Query;
 
 @Dao
 public class CinemaHallDaoImpl implements CinemaHallDao {
+    private static Logger logger = Logger.getLogger(CinemaHallDaoImpl.class);
```
Comment (Martynice) [inline src/main/java/movie/tickets/dao/impl/CinemaHallDaoImpl.java]: Same here and in other classes

File: src/main/java/movie/tickets/Main.java [hunk 3]
```diff
@@ -67,24 +69,24 @@ public static void main(String[] args) {
...
         User user1 = authenticationService.register("user1@gmail.com", "pass");
         User user2 = authenticationService.register("user2@gmail.com", "pass");
-        System.out.println(userService.findByEmail("user1@gmail.com").toString());
+        logger.info(userService.findByEmail("user1@gmail.com").toString());
         try {
-            System.out.println(authenticationService.login("user2@gmail.com", "pass").toString());
+            logger.info(authenticationService.login("user2@gmail.com", "pass").toString());
         } catch (AuthenticationException e) {
             e.getMessage();
         }
```
Comment (dmytro-hryhoruk) [inline src/main/java/movie/tickets/Main.java]: let's log error here with message

Comment (tarkaiev) [reply src/main/java/movie/tickets/Main.java]: done

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**